### PR TITLE
Kool CI Site Copy Fixes

### PR DESCRIPTION
**Pull request by indiegenus**
Reviewed the site copy and made some fixes, mostly to improve English grammar.

### CODE REVIEW

File: laravel/gitlab/index.html [hunk 1]
```diff
@@ -118,7 +118,7 @@ <h3>Gitlab CI configuration for a Laravel application running with Kool.dev</h3>
   php-security-checker: kool run phpqa security-checker security:check composer.lock
   phpunit: kool exec app ./vendor/bin/phpunit
   start:chrome: kool docker -- --name="chromedriver" --network=$KOOL_GLOBAL_NETWORK -d --shm-size=256M fireworkweb/chromedriver:latest
-  stop:chrome: docker stop chromedriver
+  stop:chrome: kool docker stop chromedriver
```
Comment (indiegenus) [inline laravel/gitlab/index.html]: @dbpolito @fabriciojs - in particular, confirm `kool` was missing here.

**Comment by indiegenus** [comment]
BTW, what a pleasure it was to onboard this project so easily and quickly, using Kool! To get to work, I simply did the following, and was up and running in no time at all.

1. Forked the CI project
2. Started Docker Desktop for Mac
3. `git clone git@github.com:indiegenus/ci.git`
4. `cd ci` and `kool start`
5. Opened `http://localhost`
6. Boom!

File: laravel/gitlab/index.html (hunk 1 above)
Comment (fabriciojs) [reply laravel/gitlab/index.html]: @indiegenus actually here we cannot use `kool`. `kool docker` is an abstraction straight to `docker run` so we cannot use for `docker stop`, like in this case. So we need to keep using `docker` bin directly.

**Review by fabriciojs — CHANGES_REQUESTED** [review]
Awesome, thanks Steve!

About that one point of attention, indeed we need to keep it back as it was.

**Comment by indiegenus** [comment]
@fabriciojs - I reverted the change to `docker stop`, and removed `kool`.

**Review by fabriciojs — APPROVED** [review]

**Pull request merged**